style.css

```diff
@@ -61,19 +81,99 @@ h2 {
   background: rgb(156, 156, 156);
 }
 
-.each-book {
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
-}
-
 .book-title {
   font-family: "lato", sans-serif;
   font-size: 14px;
 }
 
 .remove-button {
   width: fit-content;
-  padding: 6px;
-  border: 5px solid black;
+  height: 40px;
+  border: 3px solid black;
+}
+
+.remove-button:hover {
+  background-color: rgb(223, 221, 221);
+}
+
+.remove-button:active {
+  background-color: white;
+}
+
+/* Add Book Section */
+.add-book {
+  display: none;
+  flex-direction: column;
+  align-items: center;
+}
+
+#form {
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
+}
+
+.form-input {
+  border: 4px solid black;
+  padding: 0.5rem;
+}
+
+#submit {
+  padding: 0.5rem 3rem;
+  font-weight: bold;
+  align-self: end;
+  border: 4px solid black;
+  background-color: white;
+}
+
+#submit:hover {
+  background: rgb(180, 176, 176);
+  color: white;
+}
+
+#submit:active {
+  background: rgb(121, 119, 119);
+  color: white;
+}
+
+/* Contact info */
+.contact-info {
+  margin: auto;
+  width: 70%;
+  display: none;
+  flex-direction: column;
+}
```

Comment:
- [ ] [REQUIRED] Great job implementing the `contact-info` class, however, kindly take note that the contact wireframe on the template provided is aligned to the center of the page. Kindly implement this in order to fully meet up with the project requirement.

### Your Implementation
![Screenshot from 2022-03-16 23-43-12](https://user-images.githubusercontent.com/58692590/158703487-eb513495-8896-461a-9f73-ca0aae7dc95b.png)
### Template Provided
![Screenshot from 2022-03-16 23-39-10](https://user-images.githubusercontent.com/58692590/158703575-509f13da-be9a-4ea9-9e03-e49a3e424c80.png)